doc/openocd.texi

```diff
@@ -10716,6 +10716,28 @@ Perform a 32-bit DMI read at address, returning the value.
 Perform a 32-bit DMI write of value at address.
 @end deffn
 
+@subsection RISC-V Trigger Commands
+
+The RISC-V Debug Specification defines several trigger types that don't map
+cleanly onto OpenOCD's notion of hardware breakpoints. These commands let you
+set those triggers directly. (It's also possible to do so by writing the
+appropriate CSRs.)
+
+@deffn {Command} {riscv itrigger set} [@option{m}] [@option{s}] [@option{u}] [@option{vs}] [@option{vu}] [@option{nmi}] mie_bits
+Set a type 4 trigger on the current target. @option{m}, @option{s}, @option{u},
+@option{vs}, and @option{vu} control which execution modes the trigger fires in.
+If [@option{nmi}] is passed then the trigger will on fire non-maskable
+interrupts in those modes.  mie_bits controls which interrupts the trigger fires
```

Comment:
Minor things: 

* Should the "mie_bits" option be formatted like the others?
* You may also mention that the trigger action is to halt the hart (enter debug mode).

```suggestion
interrupts in those modes and cause the hart to halt.  @option{mie_bits} controls which interrupts the trigger fires
```

Reply:
Ah, I found the correct formatting for mie_bits.
I'm updating it to mention that firing means halting, but with slightly different wording than you suggested.